prescription-sync-job/overlays/ocp4-stage/thoth-notification.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: chat-notification-success-
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+        - name: chat-notification
+          image: registry.access.redhat.com/ubi8/ubi
+          command:
+            - "curl"
+            - "-X"
+            - "POST"
+            - "-H"
+            - "Content-Type: application/json; charset=UTF-8"
+            - "-d"
+            - "{'text':'I have successfully synchronized *prescription-sync* components to *OCP4-STAGE*, see <https://argocd-server-aicoe-argocd.apps.ocp4.prod.psi.redhat.com/applications/test-thoth-prescription-sync|ArgoCD UI>'}"
```

Comment:
```suggestion
            - "{'text':'I have successfully synchronized *prescription-sync* components to *OCP4-STAGE*, see <https://argocd-server-aicoe-argocd.apps.ocp4.prod.psi.redhat.com/applications/stage-thoth-prescription-sync|ArgoCD UI>'}"
```

Reply:
I copied templates from graph-backup, they might require similar changes as I kept this part untouched.